tests/PricingFactoryTest.php

```diff
@@ -1,60 +1,70 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit;
```

Comment:
Namespace should be `Blesta\Pricing\Tests;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).